### PR TITLE
Web-Console: add go to task for datasources

### PR DESCRIPTION
**Pull request by mcbrewster**
localhost_18081_unified-console html (19)

### Description
- added the ability to set a default datasource in the goToTask function of console application. 
- added a got to tasks button to datasources actions
- updated tasks view to filter datasources in when user types in either the supervisors or tasks table

This PR has:
- [x] been self-reviewed.
   - [ ] using the [concurrency checklist](https://github.com/apache/incubator-druid/blob/master/dev/code-review/concurrency.md) (Remove this item if the PR doesn't have any relation to concurrency.)
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. Linked related entities via Javadoc links.
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths.
- [ ] added integration tests.
- [ ] been tested in a test Druid cluster.

### CODE REVIEW

File: web-console/src/views/task-view/tasks-view.tsx [hunk 1]
```diff
@@ -208,7 +210,16 @@ ORDER BY "rank" DESC, "created_time" DESC`;
       showTerminateAllSupervisors: false,
 
       tasksLoading: true,
-      taskFilter: props.taskId ? [{ id: 'task_id', value: props.taskId }] : [],
+      taskFilter: props.taskId
+        ? props.datasourceId
+          ? [
+              { id: 'task_id', value: props.taskId },
```
Comment (mcbrewster) [inline web-console/src/views/task-view/tasks-view.tsx]: @vogievetsky I realize this is probably a bed way to do this, where should this actually live

**Comment by fjy** [comment]
I wish I could see number of tasks running as a column in the table, then click into those tasks to reach the tasks views.

File: web-console/src/views/task-view/tasks-view.tsx (hunk 1 above)
Comment (vogievetsky) [reply web-console/src/views/task-view/tasks-view.tsx]: probably in the constructor

**Comment by vogievetsky** [comment]
@fjy that totes makes sense but I think out of scope if this particular PR. It is coming though.

File: web-console/src/views/task-view/tasks-view.tsx [hunk 2]
```diff
@@ -528,6 +544,15 @@ ORDER BY "rank" DESC, "created_time" DESC`;
               ? 'No supervisors'
               : supervisorsError || ''
           }
+          filtered={supervisorFilter}
+          onFilteredChange={filtered => {
+            const datasources = filtered.filter(filter => filter.id === 'datasource')[0];
```
Comment (vogievetsky) [inline web-console/src/views/task-view/tasks-view.tsx]: small nit: `x.filter(...)[0]` is a less good form of `x.find(...)` also the variable name should be singular (like `datasourceFilter`)

**Comment by vogievetsky** [comment]
Looks great, thank you for addressing my feedback 👍

**Pull request merged**